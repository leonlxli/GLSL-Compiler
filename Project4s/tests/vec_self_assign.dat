funct: main
gin: v3a, vec3, 5.0, 6.0, 7.0
gin: v3b, vec3, 10.0, 20.0, 10.0
