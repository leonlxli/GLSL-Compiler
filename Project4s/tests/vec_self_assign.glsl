vec3 v3a; // 5.0, 6.0, 7.0
vec3 v3b; //10.0, 20.0, 10.0

float main() {
    v3a += 1.0; // 6.0, 7.0, 8.0
    v3a *= v3b; // 50.0, 140.0, 80.0
    return v3a.y;
}
