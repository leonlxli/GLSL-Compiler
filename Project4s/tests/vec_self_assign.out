Result: 1.400000e+02
